Gemfile

```diff
@@ -48,7 +48,11 @@ group(:omnibus_package) do
   gem "chef-provisioning-aws", ">= 3.0.2", group: :provisioning
   gem "chef-provisioning-fog", ">= 0.26.1", group: :provisioning
   gem "chef-vault", ">= 3.3.0"
-  gem "chef", "= 14.3.37"
+  # Expeditor manages the version of chef released to Rubygems. We only release 'stable' chef
+  # gems to Rubygems now, so letting this float on latest should always give us the latest
+  # stable release. May have to re-pin around major version bumping time, or during patch
+  # fixes.
+  gem "chef"
```

Comment:
We should probably make this > 14 to avoid weird deps that would drag chef down. It's why we >= most of the stuff here.

Reply:
Will do - gotta get `fog-openstack` pinned too before we merge this